azure-pipelines-gcp-integration.yml

```diff
@@ -74,10 +74,10 @@ stages:
         strategy:
           matrix:
             expectations_cfe:
```

Comment:
May as well rename these labels in the matrix from expectations_cfe to expectations_v3_api and expectations to expectations_v2_api

Reply:
great catch @kenwade4 thank you. Added suggestion